### PR TITLE
Test other update endpoints

### PR DESCRIPTION
**Pull request by amstone326**
Re-do of https://github.com/dimagi/commcare-ui-tests/pull/54, but without the test for the latest-saved-state endpoint. Clark and I decided that given the complications of testing it in calabash, it made sense to test that one via manual QA (I've added a sheet to the master test plan for that: https://docs.google.com/spreadsheets/d/11OSH0pI4HbvbHiDWNrIrXvDwVzVnhcBibAshOIWmgng/edit#gid=760951263).

### CODE REVIEW

File: features/app_update.feature [hunk 1]
```diff
@@ -65,6 +71,20 @@ Scenario: Ensure app update can be obtained from HQ
...
 
+    # Change the update endpoint to "Latest build" and update again 
+    Then I go back to the home screen
+    Then I select "Settings" menu item
+    Then I touch the "Update Options" text
+    Then I touch the "Latest build" text
+    Then I go back to the home screen
+    Then I select "Update App" menu item
+    Then I wait to see "Update to version"
+    Then I see the text "version 12"
```
Comment (nsclark) [inline features/app_update.feature]: Should this say `version 19`? Basing that off the latest version on the ui-test domain: https://www.commcarehq.org/a/ui-tests/apps/view/0841e3049fd00508782eb4f996e36611/releases/

**Comment by amstone326** [comment]
@nsclark Thank you for catching that!! Is ok now.

**Comment by nsclark** [comment]
👍 looks good

**Comment by amstone326** [comment]
@nsclark Sweet, feel free to merge then!

**Comment by wpride** [comment]
@amstone326 FWIW HQ Repositories take "Approved" to mean "You can merge this yourself" which I think makes sense

**Comment by amstone326** [comment]
@wpride What is the point of that extra step though? i.e. if a reviewer is going to mark it as approved why would they not just merge it?

**Comment by nsclark** [comment]
@amstone326 @wpride Cause I'm a noob? 😄  thanks for merging that Will.

**Comment by amstone326** [comment]
Haha @nsclark not what I meant! Just curious why HQ would use that as a standard workflow

**Comment by wpride** [comment]
@amstone326 On HQ its mostly a thing because the tests can take hours/days to run, so you want to give the PR owner permission to merge once the tests pass - not applicable here. However I think it can also be like "I approve of this PR but don't want the responsibility of merging" which maybe applies

**Pull request merged**